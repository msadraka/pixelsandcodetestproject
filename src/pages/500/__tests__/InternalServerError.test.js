import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import InternalServerError from '../index';

describe('<InternalServerError />', () => {
  it('renders', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <InternalServerError />
        </MemoryRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
