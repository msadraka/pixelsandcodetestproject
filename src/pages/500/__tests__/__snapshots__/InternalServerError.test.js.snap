// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InternalServerError /> renders 1`] = `
<div
  className="errorstyle___default-sc-14ginyp-0 ecTVbD"
>
  <p>
    Oops, Internal Server Error
  </p>
</div>
`;
